new-feature.txt

```diff
@@ -0,0 +1 @@
+this features was created in the feature-1 branch
```

Comment:
its really good that you have decided to add this line of code

Reply:
thanks for this comment

Reply:
teste